storage/miner.go

```diff
@@ -203,6 +203,16 @@ func (m *Miner) Run(ctx context.Context) error {
 	return nil
 }
 
+func getDefaultSectorExpirationExtension(cfg sealing.GetSealingConfigFunc) abi.ChainEpoch {
```

Comment:
I think we should wire the config getter into the precommit policy, and use the up-to-date value on each call to get sector expiration (also do the validation there).